src/test/java/com/flightstats/hub/dao/aws/S3VerifierUnitTest.java

```diff
@@ -1,7 +1,7 @@
 package com.flightstats.hub.dao.aws;
 
```

Comment:
It would be nice to remove unit from class name. Eventually all integrationtests will be in integrationTest folder which i am currently working on.

Reply:
I think I'll tackle that once  we have that package/directory. I'd also like to keep integration tests (testing groups of classes) separate from functional/system tests, which I think of as being more black-box. That way we can draw better lines about what dependencies we have access to in tests. I think the system tests should only be able to access the hub client and its models but shouldn't access most of the hub code itself.